portal/static/js/main.js

```diff
@@ -1280,6 +1296,76 @@ var fillContent = {
             });
         };
     },
+    "notifications": function(data) {
+        if (data && data.notifications) {
+            var notifications = [];
+            var notificationText = "";
+            (data.notifications).forEach(function(notice) {
+                notificationText += "<div class='notification' data-id='" + notice.id + "'>" + i18next.t(notice.content) + "</div>";
+            });
+            if (hasValue(notificationText)) {
+
+                var infoText = "<div class='notification-info'><i class='fa fa-info-circle' aria-hidden='true'></i>";
+
+                if (data.notifications.length > 1) {
```

Comment:
@achen2401 Just a heads up, we'll likely have to better implement [plural forms](https://www.i18next.com/plurals.html)